package.json

```diff
@@ -24,13 +24,14 @@
   "devDependencies": {
     "eslint": "^7.9.0",
     "eslint-config-origami-component": "^2.1.0",
-    "origami-ci-tools": "^2.1.0",
+    "origami-ci-tools": "3.0.0-beta.4",
+    "remark": "^12.0.1",
+    "remark-lint": "^7.0.1",
+    "remark-preset-lint-origami-component": "^2.0.0-beta.1",
+    "stylelint": "^13.7.0",
     "stylelint-config-origami-component": "^1.0.4",
     "typescript": "^4.0.5"
   },
-  "publishConfig": {
-    "registry": "https://origami-npm-registry-prototype.herokuapp.com"
-  },
   "scripts": {
     "build": "tsc src/**/*.js main.js --declaration --allowJs --emitDeclarationOnly --outDir types"
```

Comment:
If we are going to have a build step, we need to add a `version` npm run-script which runs the build step and adds the changed files to the git repository, something like this:
```
"version": "npm run build && git add ./types"
```

Reply:
o-brand has no js 🔥